taxonomies/categories.txt

```diff
@@ -4747,10 +4755,19 @@ ru:Кофе растворимый сублимированный, Раство
 
 <en:Instant beverages
 <en:Plant-based beverages
-en:Instant chicory
+en:Instant chicory with water
```

Comment:
we should keep "Instant chicory" : that's what the product is.
it's fine to associate the CIQUAL category to it.